javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,160 @@
+// Project 1.  Smart-ease - Goes Global!
+
+function getFullname(firstName, surName, userFormalName) {
+    if (userFormalName) {
+        return "Lord" + " " + firstName + surName
+    } else {
+        return firstName + surName;
+    }
+};
+
+let fullname1 = getFullname("Benjamin", " Hughes", true);
+let fullname2 = getFullname("Benjamin", " Hughes", false);
+console.log(fullname1);
+console.log(fullname2);
+
+
+/* Question from assignement - But what if the person is a woman? 
+Describe how you would fix the getFullname so it also works for women
+*/
+
+/* Honestly I have no idea how it should be fixed in JavaScript without fixing
+ it before in HTML with radio buttons which specify gender. It's possible to
+ create a new function adding female prefix to fullname but I don't know how system 
+ can differntiate if it's a female or a male name.
+*/
+
+
+
+
+//Project 2. Event application
+
+const days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+
+
+function getEventWeekday(eventDay) {
+    let today = new Date();
+    let newDay = today.getDay();
+    eventDayNumber = (newDay + eventDay) % 7;
+    return eventDayNumber;
+}
+console.log(getEventWeekday(9));
+console.log(getEventWeekday(3));
+
+
+
+//Project 3. Weather wear
+
+function suggestClothes(temp) {
+
+    if (temp < 0) {
```

Comment:
If you call the function with _0_ then the function returns nothing.
```suggestion
    if (temp <= 0) {
```